packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -143,6 +143,28 @@ const getCollaterals = async ({ state }) => {
   );
 };
 
+const getLiquidationConfig = directorParamManager => ({
+  install: directorParamManager.getInstallation(LIQUIDATION_INSTALL_KEY),
+  terms: directorParamManager.getUnknown(LIQUIDATION_TERMS_KEY),
+});
+
+const watchGovernance = (govParams, vaultManager, oldInstall, oldTerms) => {
+  const subscription = govParams.getSubscription();
+  observeIteration(subscription, {
+    updateState(_paramUpdate) {
+      const { install, terms } = getLiquidationConfig(govParams);
+      if (install === oldInstall || keyEQ(terms, oldTerms)) {
```

Comment:
I think you don't want to block updates when the installation is the same. The only way to change the terms is to start a new instance with different terms.
```suggestion
      if (install === oldInstall && keyEQ(terms, oldTerms)) {
```

Reply:
Yep. That was a bug. It should have been &&